grc/core/blocks/embedded_python.py

```diff
@@ -81,17 +81,19 @@ class EPyBlock(Block):
     label = 'Python Block'
     documentation = {'': DOC}
 
-    parameters_data = [dict(
+    parameters = [dict(
         label='Code',
         id='_source_code',
         dtype='_multiline_python_external',
         value=DEFAULT_CODE,
         hide='part',
     )]
-    inputs_data = []
-    outputs_data = []
+    inputs = []
```

Comment:
I have some doubt now if removing '_data' from inputs_data and outputs_data is correct thing to do.